clusterloader2/pkg/config/cluster.go

```diff
@@ -40,6 +40,8 @@ type ClusterConfig struct {
 	MasterName                 string
 	KubemarkRootKubeConfigPath string
 	DeleteStaleNamespaces      bool
+	IsSSHSupported             bool
```

Comment:
We should make it clear that it's the SSH to master that is not supported (e.g. in GKE you can still ssh into nodes). 
`IsSSHToMasterSupported`?